NOTICE

```diff
@@ -0,0 +1,5 @@
+Oppia
+Copyright 2014-2019 The Oppia Authors
```

Comment:
Are we expecting to have a period at the end of this line (not sure!)?

(Similar to what we have in all of the files `# Copyright 2014 The Oppia Authors. All Rights Reserved.`)

Reply:
Ignore this comment, I went through the link provided in the top. Thanks!

Reply:
I had exactly that thought process :)

Thanks!